zephyr/module.yml

```diff
@@ -59,3 +59,55 @@ blobs:
     license-path: zephyr/blobs/license/LA_OPT_NXP_Software_License.txt
     url: https://github.com/nxp-mcuxpresso/mcux-sdk-middleware-ieee_802.15.4/raw/MCUX_2.16.100/bin/mcxw71/mcxw71_nbu_ble_15_4_dyn_1_0_17_2.sb3
     description: "BLE Controller and IEEE 802.15.4 PHY combo firmware for MCXW71 boards"
+
+#For NW61x
+  - path: nw61x/sd_nw61x.bin.se
```

Comment:
@abhinavnxp I believe this is the PR to ask TSC approval for this blob addtion. https://github.com/zephyrproject-rtos/zephyr/issues/83905

Can you please confirm?